### ISSUE TITLE
Is that possible to generate s3 credentials programmatically 

### ISSUE THREAD

**Issue by ghost** [chrislusf/weed-fs#5331]
To generate and store them in a database. I don't think that configuration file with s3 keys will scale well. Is there a solution for that out there already? Or do I have to write something of my own?

Thank you in advance.

**Comment by chrislusf** [comment]
Good point. We need to split the configuration file to one-bucket-one-file.
